apps/meteor/app/lib/server/lib/notifyUsersOnMessage.ts

```diff
@@ -137,7 +137,6 @@ async function updateUsersSubscriptions(message: IMessage, room: IRoom): Promise
 	]);
 
 	// update subscription of the message sender
-	await Subscriptions.setAsReadByRoomIdAndUserId(message.rid, message.u._id);
 	const setAsReadResponse = await Subscriptions.setAsReadByRoomIdAndUserId(message.rid, message.u._id);
 	if (setAsReadResponse.modifiedCount) {
```

Comment:
btw, this `modifiedCount` will always be truthy for existing subscriptions, because `setasReadyByRoomIdAndUserId` will always set `ls: new Date()`